test/cypress/integration/group-contributions.spec.js

```diff
@@ -0,0 +1,533 @@
+const userId = Math.floor(Math.random() * 10000)
```

Comment:
NOTE: This file is not new, I just renamed `group-payments.spec.js` to `group-contributions.spec.js` and did not modify it at all.
Then, I created a new `group-payments.spec.js` to write the actual tests related to payments. (where we also override the time `Date` with `cy.clock`)